src/components/Typography/styled.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
+
+export const Wrap = styled.div`
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  background-color: ${(props) => props.theme.colors.white};
+`;
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  background-color: ${(props) => props.theme.colors.whiteSmoke};
+  width: 90vw;
+  height: 220vh;
```

Comment:
You set 220vh. But if you will want to increase the rows of your table you will need to increase this value. Think about a more universal solution